### ISSUE TITLE
[Bug]: When chatting in a knowledgebase, it first shows the answer to the last question

### ISSUE THREAD

**Issue by linxiaohui** [infiniflow/ragflow#2652]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Branch name

dev

### Commit ID

docker tag dev

### Other environment information

```Markdown
Linux with docker-compose


REPOSITORY                                      TAG                            IMAGE ID       CREATED        SIZE
infiniflow/ragflow                              dev                            4d406d8bfecd   3 days ago     19.2GB
mysql                                           8.0.39                         f5da8fc4b539   2 months ago   573MB
redis                                           7.2.4                          9b38108e295d   5 months ago   116MB
quay.io/minio/minio                             RELEASE.2023-12-20T01-00-02Z   73f9f0b5b015   9 months ago   151MB
docker.elastic.co/elasticsearch/elasticsearch   8.11.3                         792fab0c0bd8   9 months ago   1.43GB
```
```


### Actual behavior

After successfully setting up the knowledge base, when chatting, the first one is OK, but since the 2nd question, when enter the question, it show the answer to the last question immediately, and after a while it begins to show the answer of the question.

### Expected behavior

do not show the response of last question

### Steps to reproduce

```Markdown
Ask more than 2 questions
```

### Additional information

the console of browser shows
```
logic-hooks.ts:274 SyntaxError: Unexpected token 'd', "data:{"ret"... is not valid JSON
（匿名） @ logic-hooks.ts:274
```

**Comment by KevinHuSh** [comment]
Already fixed.

**Comment by generatemotion** [comment]
I have some the issue in master, and cannot find the file with fixed #2652,could you show me the details about the #2652
